telosnetwork/open-block-explorer
# Add token icons

# Fixes #461

## Test Scenarios:
- Go to my [personal account token page](https://461-add-token-icons.open-block-explorer.pages.dev/account/viterbotelos?tab=tokens)
- All tokens should have their token icon
- Go home transaction table section
- try to filter by the token. Type just the letter "t"
- All the tokens shown should add an icon, symbol, and contract name.

## Draft
This PR is still in the draft because it is blocked until [this PR](https://github.com/telosnetwork/token-list/pull/38) gets approved in the token-list repo. Until then you can try this branch which fetches the token lists from [a fork ](https://github.com/Viterbo/token-list/tree/main) I have in my personal namespace.

## Screenshots
![image](https://user-images.githubusercontent.com/4420760/221383761-bc8080c3-54aa-4c37-bf0d-08269f9606f9.png)

![image](https://user-images.githubusercontent.com/4420760/221383785-fc0ebb1c-a0d4-4389-aaed-a0bda8ece349.png)
